week05/errors_test.go

```diff
@@ -0,0 +1,176 @@
+package demo
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestErrTableNotFoundError(t *testing.T) {
+	t.Parallel()
+
+	tn := "testtable"
+	testError := ErrTableNotFound{
+		table:		tn,
+	}
+
+	exp := "table not found " + tn
+	act := testError.Error()
```

Comment:
We never assert against an error message. Please re-read the sections on `As` and `Is`, as well as the helper functions in the available code, for the proper way of making error assertions.

Reply:
I do get that we should never assert against an error message, but here I am testing a function that returns a string. Testing the expected output is here nothing more or less than testing whether the returned string matches the expected string imho.

Reply:
You're correct, sorry. I didn't notice the context in which it was being used.